package.json

```diff
@@ -16,7 +16,7 @@
     "eslint": "^8.56.0",
     "prettier": "^3.2.5",
     "rimraf": "^5.0.7",
-    "turbo": "1.13.3"
+    "turbo": "2.0.5"
```

Comment:
This change applies to the whole monorepo and don't suggest we include it in this PR. Can we revert this change?

It would be great if you could create a Shortcut story for the goal of upgrading to turbo 2.x. Please note there's a whole guide of steps that we should follow when we implement this change: https://turbo.build/repo/docs/crafting-your-repository/upgrading

Reply:
Seems like we're already using turbo 2.0.5 on main 🤔

Reply:
@eduramme Ah if this change is already on main, then I didn't mean to create confusion with my feedback

Reply:
@eduramme ... but I still understand we need to follow that upgrade process properly that I linked above and I'm not sure we've followed that yet. Therefore please let me know how you suggest we proceed. You're welcome to create a new shortcut story for the turbo upgrade as one option for how we proceed.